subgraphs/compound-forks/protocols/iron-bank/src/mapping.ts

```diff
@@ -299,6 +338,157 @@ export function handleTransfer(event: Transfer): void {
   );
 }
 
+export function handleStakingRewardsCreated(
+  event: StakingRewardsCreated
+): void {
+  log.info(
+    "[handleStakingRewardsCreated]StakingReward contract {} created for token {} at tx {}",
+    [
+      event.params.stakingRewards.toHexString(),
+      event.params.stakingToken.toHexString(),
+      event.transaction.hash.toHexString(),
+    ]
+  );
+
+  StakingRewardsTemplate.create(event.params.stakingRewards);
+}
+
+export function handleStaked(event: Staked): void {
+  const rewardContract = StakingRewardsContract.bind(event.address);
+  const marketID = rewardContract.getStakingToken().toHexString();
```

Comment:
should use some sort of error handling for contract calls. I think the `getOrElse()` in compound forks would work nicely here